example/nuxt.config.js

```diff
@@ -1,5 +1,30 @@
+import { join } from 'path'
+
+import { watch } from 'chokidar'
+import { mkdist } from 'mkdist'
+
 export default {
   modules: [
     '../src/module.ts'
-  ]
+  ],
+  hooks: {
+    async 'modules:before' () {
+      if (process.env.NODE_ENV === 'test') {
+        return
+      }
+      // This is for developing templated files
+      // and ensures we get live-reloading in our Nuxt example
+      // whilst editing TypeScript sources.
+      const path = join(__dirname, '../src/templates')
```

Comment:
Dependencies should be watched by nuxt and bundler. We should avoid such boilerplate inside module template

Reply:
Some other options I considered and rejected:

1. Add boilerplate within `module.ts` instead.
2. Insert `.ts` plugin directly when developing - but this doesn't work atm with `@nuxt/typescript-build`
3. watch mode for `mkdist` (but this is clunkier and it's possible to get out of sync without realising it)

If we don't want this boilerplate then likely the best approach is to add transpile support for `addPlugin` within `@nuxt/typescript-build`. Thoughts?